src/main/java/uk/co/onsdigital/discovery/metadata/api/dto/legacy/DataSet.java

```diff
@@ -12,13 +13,13 @@
 @JsonInclude(JsonInclude.Include.NON_EMPTY)
 public class DataSet {
 
-    private String id;
-    private String s3URL;
-    private String title;
-    private String url;
-    private String metadata;
-    private List<DimensionMetadata> dimensions = Collections.emptyList();
-    private String dimensionsUrl;
+    protected String id;
```

Comment:
These are protected as `DimensionalDataSet` extends this class and uses these variables.